garrysmod/gamemodes/terrortown/gamemode/vgui/sb_row.lua

```diff
@@ -82,6 +82,16 @@ function GM:TTTScoreboardColorForPlayer(ply)
    return namecolor.default
 end
 
+function GM:TTTScoreboardRowColorForPlayer(ply)
+   if ply:IsTraitor() then
+      return Color(255, 0, 0, 30)
+   elseif ply:IsDetective() then
+      return Color(0, 0, 255, 30)
+   end
+
+   return Color(0, 0, 0, 0)
+end

```

Comment:
This is creating new Color objects for every single row that's rendered. It would be more efficient to create the colors once and just return them in this function. Same as is being done with `namecolor` right above here.